---
layout: single
author_profile: true
---

{% include base_path %}

<h1 class="page__title">Research Interests</h1>
<h2>1. Epigenetic regulation of gene expression</h2>
<p>Gene regulation processes control the temporal and spatial expression of genes, which leads to the derivation of a variety of tissues and cell types from the same genome. The concept of epigenetics is that functionally relevant changes to the genome may not involve a change in the nucleotide sequence. Examples of mechanisms that produce such changes are DNA methylation and histone modification, each of which regulates how genes are expressed without altering the underlying DNA sequence but altering the interactions between cis- and trans-acting elements. We focus on annotating these elements and the non-coding genomic regions by interpreting the epigenentic signatures of them.</p>

<h2>2. Cancer genomics</h2>
<p>Cancer is a disease of the genome. The past decade has seen breath-taking advances in the characterization of point mutations and structural alterations in a wide range of cancers; all thanks to the next-generation sequencing technologies. Increasingly, the complete genome sequences of a large number of cancer types are being obtained, providing us with a comprehensive view of cancer development. We focus on interpreting these cancer genomic data using advanced statistical and machine-learning methods to assess their application on clinical treatment.</p>

<h1 class="page__title">Publications</h1>

<p>Zhang, Z., Huang, K., Gu, C., Zhao, L., Wang, N., Wang, X., … Meng, Y. (2016). Molecular Subtyping of Serous Ovarian Cancer Based on Multi-omics Data. Scientific Reports, 6, 26001. <a href="http://doi.org/10.1038/srep26001">Full text</a></p>
<p>Lu, Y., Shan, G., Xue, J., Chen, C., & Zhang, C. (2016). Defining the multivalent functions of CTCF from chromatin state and three-dimensional chromatin interactions. Nucleic Acids Research, gkw249. <a href="http://doi.org/10.1093/nar/gkw249">Full text</a></p>
<p>Gao, Y., Lu, Y., Yi, J., Li, Z., Gao, D., Yu, Z., … Zhang, C. (2016). A Genome-Wide mRNA Expression Profile in Caenorhabditis elegans under Prolonged Exposure to 1750MHz Radiofrequency Fields. Plos One, 11(1), e0147273. <a href="http://doi.org/10.1371/journal.pone.0147273">Full text</a></p>
<p>Lu, Y., Qu, W., Shan, G., & Zhang, C. (2015). DELTA: A Distal Enhancer Locating Tool Based on AdaBoost Algorithm and Shape Features of Chromatin Modifications. Plos One, 10(6), e0130622. <a href="http://doi.org/10.1371/journal.pone.0130622">Full text</a></p>
<p>Xie, P., Zhang, M., He, S., Lu, K., Chen, Y., Xing, G., … Zhang, L. (2014). The covalent modifier Nedd8 is critical for the activation of Smurf1 ubiquitin ligase in tumorigenesis. Nat Commun, 5. Retrieved from <a href="http://dx.doi.org/10.1038/ncomms4733">Full text</a></p>
<p>Lu, Y., Qu, W., Min, B., Liu, Z., Chen, C., & Zhang, C. (2014). Modelling epigenetic regulation of gene expression in 12 human cell types reveals combinatorial patterns of cell-type-specific genes. IET Systems Biology, 8(3), 104–15. <a href="http://doi.org/10.1049/iet-syb.2013.0042">Full text</a></p>
<p>Lu, Y., Zhang, Y., Hang, X., Qu, W., Lubec, G., Chen, C., & Zhang, C. (2013). Genome-wide computational identification of bicistronic mRNA in humans. Amino Acids, 44(2), 597–606. <a href="http://doi.org/10.1007/s00726-012-1380-1">Full text</a></p>
<p>Li, W., Zhang, Y., Liu, Y., Yue, F., Lu, Y., Qiu, H., … Zhang, C. (2012). In vitro kinetic evaluation of the free radical scavenging ability of propofol. Anesthesiology, 116(6), 1258–66. <a href="http://doi.org/10.1097/ALN.0b013e3182567dcc">Full text</a></p>
<p>Qu, W., Zhou, Y., Zhang, Y., Lu, Y., Wang, X., Zhao, D., … Zhang, C. (2012). MFEprimer-2.0: a fast thermodynamics-based program for checking PCR primer specificity. Nucleic Acids Research, 40(Web Server issue), W205–8. <a href="http://doi.org/10.1093/nar/gks552">Full text</a></p>
<p>Qu, W., Ren, C., Li, Y., Shi, J., Zhang, J., Wang, X., … Zhang, C. (2011). Reliability analysis of the Ahringer Caenorhabditis elegans RNAi feeding library: a guide for genome-wide screens. BMC Genomics, 12(1), 170. <a href="http://doi.org/10.1186/1471-2164-12-170">Full text</a></p>
<p>Zhou, Y., Qu, W., Lu, Y., Zhang, Y., Wang, X., Zhao, D., … Zhang, C. (2011). VizPrimer: a web server for visualized PCR primer design based on known gene structure. Bioinformatics (Oxford, England), 27(24), 3432–4. <a href="http://doi.org/10.1093/bioinformatics/btr582">Full text</a></p>
<p>Lu, Y., Zhou, Y., Qu, W., Deng, M., & Zhang, C. (2011). A Lasso regression model for the construction of microRNA-target regulatory networks. Bioinformatics (Oxford, England), 27(17), 2406–2413. <a href="http://doi.org/10.1093/bioinformatics/btr410">Full text</a></p>
<p>Shen, Z., Qu, W., Wang, W., Lu, Y., Wu, Y., Li, Z., … Zhang, C. (2010). MPprimer: a program for reliable multiplex PCR primer design. BMC Bioinformatics, 11, 143. <a href="http://doi.org/10.1186/1471-2105-11-143">Full text</a></p>

{% include paginator.html %}
